src/mgr/PyState.cc

```diff
@@ -152,7 +152,24 @@ ceph_send_command(PyObject *self, PyObject *args)
       return nullptr;
     }
   } else if (std::string(type) == "pg") {
-    // TODO: expose objecter::pg_command
+    pg_t pgid;
+    if (!pgid.parse(name)) {
+      delete c;
```

Comment:
this also silences a clang analyzer warning at http://people.redhat.com/bhubbard/scan-build-2017-06-21-181754-22713-1/report-863687.html#EndPath